spswarehouse/powerschool/powerschool.py

```diff
@@ -0,0 +1,437 @@
+import logging
+import time
+import os
+
+from ducttape.utils import (
+    get_most_recent_file_in_dir
+)
+
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.support.ui import WebDriverWait, Select
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.common.exceptions import NoSuchElementException
+
+ADMIN_LOGIN_PAGE_PATH = 'admin/pw.html'
+ADMIN_HOME_PAGE_PATH = 'admin/home.html'
+ADMIN_URL_SCHEME = 'https://'
+STATE_REPORTS_PAGE_PATH = 'admin/reports/statereports.html?repType=state'
+REPORT_QUEUE_REPORTWORKS_PAGE_PATH = 'admin/reportqueue/prhome.html'
+REPORT_QUEUE_SYSTEM_PAGE_PATH = 'admin/reportqueue/home.html'
+
+def log_into_powerschool_admin(driver: WebDriver, 
```

Comment:
from a packaging perspective, I wonder if this should all be a class, where the WebDriver is part of the class, rather than having to create the driver in the python file and passing it to all the functions.